04-deploying_shiny_app.Rmd

```diff
@@ -73,11 +73,12 @@ This creates a file called `packrat/packrat.lock`.  You need to ensure you have
 
 ### Scan organisation and deploy
 
-The platform automatically scans git repositories in the [moj-analytical-services](https://github.com/moj-analytical-services/) Github organisation to find repos that are ready to deploy.  It does this by checking whether repositories contains a 'magic' file that controls deployment:  `deploy.json`. The scan happens once every 24 hours (or if it is urgent it can be triggered by an admin).
+The platform automatically scans git repositories in the [moj-analytical-services](https://github.com/moj-analytical-services/) Github organisation to find repos that are ready to deploy.  It does this by checking whether repositories contains a 'magic' file (on the `master` branch) that controls deployment:  `deploy.json`. The scan happens nightly (or if it is urgent it can be triggered by an admin).
```

Comment:
the scan happens a maximum of every 24h. If there is a release in the moj-analytical-services org, then the scan is triggered by a webhook

Reply:
Thanks, I've changed it to `The scan happens every 24 hours, or sooner if a [repo in moj-analytical-services](https://github.com/moj-analytical-services) is tagged with a release on GitHub (or if it is urgent it can be triggered by an admin).`